cms/static/js/spec/video/transcripts/videolist_spec.js

```diff
@@ -1,13 +1,13 @@
 define(
     [
         'jquery', 'underscore',
+        'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpers',
         'js/views/video/transcripts/utils',
         'js/views/video/transcripts/metadata_videolist', 'js/models/metadata',
         'js/views/abstract_editor',
-        'common/js/spec_helpers/ajax_helpers',
         'xmodule'
     ],
-function ($, _, Utils, VideoList, MetadataModel, AbstractEditor, AjaxHelpers) {
+function ($, _, AjaxHelpers, Utils, VideoList, MetadataModel, AbstractEditor) {
```

Comment:
This seems like a good change to me, just curious if we have any kind of rule documented anywhere that says we put utils/external libs before local files in requirejs load lists? Could be good to add it to the developer guide.